pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Prometheus Queries in Wavefront
+keywords: query language
+sidebar: doc_sidebar
+published: true
+permalink: wavefront_prometheus.html
+summary: Run Prometheus from the Wavefront query line
+---
+
+Starting with release 2020.26, Wavefront includes support for Prometheus queries.
+* Run Prometheus queries directly from the Wavefront query line. The corresponding chart shows the information you'd expect.
+* Wavefront query language includes new functions and operators that allow you to "translate" any Prometheus query to a ts() query and use it with the rich set of existing ts() query language functions.
```

Comment:
I'm worried about this second bullet. It doesn't sound accurate and i can't figure out what the goal of this was.  What were you hoping to say here.  Now it's not accurate because although we do a translation of PromQL to ts (that happens on the backend and is transparent to the user).  Also this line seems to imply that you can mix and match PromQL and TSQL which isn't supported.

Reply:
Removed second bullet. Not useful in this context.